js/LocationSampler.ts

```diff
@@ -40,6 +44,16 @@ export default class LocationSampler {
             this.timerId = -1;
             this.running = false;
         }
+        console.log(RNFS.ExternalDirectoryPath);
+        var path = RNFS.ExternalDirectoryPath + '/' + this.measurementName + '.json';
+
+        var data = JSON.stringify({accuracy: 1, battery_start: 99, battery_end: 67, samples: this.samples});
```

Comment:
Since these default values have no use case yet, maybe we can omit them for now.